generic/tests/cfg/timedrift.cfg

```diff
@@ -27,29 +30,29 @@
                 - with_migration:
                     type = timedrift_with_migration
                     migration_iterations = 3
-                    drift_threshold = 10
-                    drift_threshold_single = 3
                 - with_reboot:
                     type = timedrift_with_reboot
                     reboot_iterations = 1
-                    drift_threshold = 10
-                    drift_threshold_single = 3
                 - with_stop:
-                    # starting from Win2012 Windows no longer relies on number of injected RTC interrupts for time keeping,
-                    # and the time drift is seen because "stop" monitor command also stops PM timer,
-                    # so this is unsupported scenario from Win2012.
-                    Windows:
-                        only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
                     type = timedrift_with_stop
                     stop_interations = 1
                     stop_time = 60
-                    drift_threshold = 10
-                    drift_threshold_single = 3
                     variants:
                         - clock_host:
-                            rtc_clock = host
+                            # starting from Win2012 Windows no longer relies on number of injected RTC interrupts for time keeping,
+                            # and the time drift is seen because "stop" monitor command also stops PM timer,
+                            # so this is unsupported scenario from Win2012.
+                            Windows:
+                                only Win2000, WinXP, Win2003, WinVista, Win7, Win2008
                         - clock_vm:
                             rtc_clock = vm
+                    variants:
```

Comment:
This variants is same as line 98, please only use one variants here to avoid duplicated work.